### PR TITLE
Redux structure

### PR DESCRIPTION
**Pull request by Racenna**

### CODE REVIEW

File: src/redux/appReducer/actions.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { getUserData } from '../authReducer/index';
+
+export const SET_INITIALIZED = 'app/SET_INITIALIZED';
+
+const setInitialized = () => ({ type: SET_INITIALIZED });
+
+export const initializeApp = () => (dispatch) => {
+  let promise = dispatch(getUserData());
+  Promise.all([promise]).then(() => {
```
Comment (worksnk) [inline src/redux/appReducer/actions.js:9]: Why you use `Promise.all` here?

File: src/redux/authReducer/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { getUserData, login, logout, setUserData } from './actions';
+import authReducer from './reducer';
+import {
+  isAuthSelector,
+  loginSelector,
+  errorMessageSelector,
+  captchaUrlSelector,
+} from './selectors';
+
+export {
```
Comment (worksnk) [inline src/redux/authReducer/index.js:10]: Why do you export them from index one by one?

**Review by worksnk — CHANGES_REQUESTED** [review]

File: src/redux/appReducer/actions.js (hunk 1 above)
Comment (Racenna) [reply src/redux/appReducer/actions.js]: Changed it

File: src/redux/authReducer/index.js (hunk 2 above)
Comment (Racenna) [reply src/redux/authReducer/index.js]: Used aggregating modules how in example on MDN

File: src/components/Dialogs/Dialogs.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import { dialogsDataSelector } from '../../selectors/dialogsSelectors';
+import { dialogsDataSelector } from '../../redux/dialogsReducer/index';
```
Comment (worksnk) [inline src/components/Dialogs/Dialogs.jsx:3]: you don't need to specify `index`. Index is loaded by default if directory specified as import path

File: src/redux/appReducer/reducer.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { SET_INITIALIZED } from './actions';
+
+const initialState = {
+  initialized: false,
+};
+
+const appReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_INITIALIZED:
```
Comment (worksnk) [inline src/redux/appReducer/reducer.js:9]: Why do you need this state in the first place?

**Review by worksnk — APPROVED** [review]

**Pull request merged**